pallets/subtensor/src/root.rs

```diff
@@ -508,7 +508,7 @@ impl<T: Config> Pallet<T> {
 
         // --- 13. Force all members on root to become a delegate.
         if !Self::hotkey_is_delegate(&hotkey) {
-            Self::delegate_hotkey(&hotkey, 11_796); // 18% cut defaulted.
```

Comment:
What is the conversion factor here? Would like to add some guardrails on the python side